src/ontology/config/ordo_exclusions.tsv

```diff
@@ -176,4 +176,5 @@ Orphanet:522514	Congenital optic disc excavation of genetic origin	FALSE	MONDO:e
 Orphanet:522562	Genetic superficial corneal dystrophy	FALSE	MONDO:excludeGroupingOriginGenetic
 Orphanet:519337	Disorder with optic nerve compression	FALSE	MONDO:excludeGroupingPhenotype
 Orphanet:522520	Syndromic genetic disorder with strabismus	FALSE	MONDO:excludeGroupingPhenotype
-Orphanet:522530	Rare genetic disorder with entropion	FALSE	MONDO:excludeGroupingPhenotype
\ No newline at end of file
+Orphanet:522530	Rare genetic disorder with entropion	FALSE	MONDO:excludeGroupingPhenotype
+Orphanet:158038	Primary hemophagocytic lymphohistiocytosis	FALSE	MONDO:excludeGrouping
```

Comment:
Just giving a face check review. I'm not familiar with how these exclusion codes work, but it does indeed seem that `MONDO:excludeGrouping` matches with what is being asked! (https://github.com/monarch-initiative/mondo/issues/7437)